src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_Backspace.cs

```diff
@@ -63,10 +63,11 @@ private void ExecuteBackspaceOrDelete(ITextView textView, Action nextHandler, bo
                     this.TextView.Caret.PositionChanged += OnCaretPositionChanged;
                 }
 
-                var triggerInfo = CompletionTriggerInfo.CreateBackspaceTriggerInfo(deletedChar);
+                var trigger = new BackspaceOrDeleteCharCompletionTrigger(deletedChar, isBackspace: !isDelete, customTags: _completionTriggerTags);
```

Comment:
I tried separating them at first and it seemed weird because every place this trigger is checked needs to check both. Now that I'm looking at it again, maybe I should go back to separating them?